CHANGELOG.md

```diff
@@ -28,6 +28,126 @@ The changelog is generated during the release process using [ChangelogLinker](ht
+- [#687] refactored AdvancedSearch filter names to Filter classes
+
+#### Fixed
+
+- [#706] fixed symfony deprecation warnings
+- [#647] fixed link to our Twitter account [@ShopsysFW] in demo data
+- [#640] Fixed cart merging after login (closes [#588]), Thanks to [@dzarda86]
+- [#628] selectboxes rollout problem for dev env
+- [#644] JsFormValidatorFactory: make sure prototype's parent is set in CollectionType
+- [#655] calculatedHidden attribute of a main variant is taken over from the hidden attribute, Thanks to [@dzarda86]
+- [#676] formatNumber filter supplied with missing orderLocale variable, Thanks to [@dzarda86]
+
+#### Removed
+
+- [#641] FormGroup: unused option "is_group_container_to_render_as_the_last_one" removed
+- [#609] remove redundant PaginationResult instantiation
+
+### [shopsys/microservice-product-search-export]
+
+#### Added
+
+- [#735] added symfony/dotenv to composer.json
+
+#### Fixed
+
+- [#637] fixed search in EAN, partno, catnum
+
+### [shopsys/project-base]
+
+#### Added
+
+- [#673] added support for custom prefixing in redis
+
+#### Changed
+
+- [#596] Trusted proxies are now configurable in parameters.yml file
+- [#703] improved frontend performance
+- [#674] Dockerignore no longer excludes docker/nginx folder
+- [#645] SVG icons in generated document
+- [#702] switched to Debian php-fpm image
+- [#740] generation of jpeg previews for php-fpm debian container
+
+#### Fixed
+
+- [#685] fix wrong variable name in flash message
+- [#660] Bootstrap: do not prematurely boot Kernel when running in console
+
+#### Removed
+
+- [#710] Removed blank space when popup is active
+
+### [shopsys/shopsys]
+
+#### Added
+
+- [#729] added scripts for monorepo maintenance into /.ci directory
+- [#623] Add Releaser to automate release process, Thanks to [@TomasVotruba]
+- [#719] New SSFW references added to readme.md
+- [#642] entity-extension.md: added warning about extending OrderItem and its descendants
+- [#705] added cookbook about basic data import
+- [#651] It's possible to add index prefix to elastic search
+- [#649] docs: added missing link to Adding an Icon into a Button cookbook
+- [#662] add missing upgrade note for [#651] when using monorepo
+- [#633] Google Cloud deploy using Terraform, Kustomize and Kubernetes
+- [#646] document behavior of less compiling
+- [#693] documentation of translations
+
+#### Changed
+
+- [#720] Updated year in license files
+- [#733] twig/twig v2.6.1 is now ignored
+- [#610] ProductOnCurrentDomainFacade: rename getPaginatedProductDetails\* methods
+- [#699] Improve upgrade instructions
+- [#708] upgrading-monorepo.md: improved instructions
+- [#709] Improved entity data objects documentation
+- [#627] model service layer removal (phase 1)
+- [#657] model service layer removal (phase 2)
+- [#672] model service layer removal (phase 4)
+- [#670] model service layer removal (phase 5)
+- [#671] model service layer removal (phase 6)
+- [#691] model service layer removal (phase 7)
```

Comment:
Please, either group this to just `[#627] model service layer removal`, or add the missing record for the manually merged `[#652]  model service layer removal (phase 3)` (and don't forget to add the link below)